input/delimiter/fallback_delimiter_test.go

```diff
@@ -1,14 +1,14 @@
-package input
+package delimiter
```

Comment:
I changed the package name and renamed the Delimiter to be the FallbackDelimiter.